app/controllers/users_controller.rb

```diff
@@ -127,6 +127,7 @@ def bulk_destroy
   # shows printable login instructions for the user
   def login_instructions
     @site_url = admin_mode? ? basic_root_url : mission_root_url
+    @config_qr = ODKConfigGenerator.new().generate_odk_config(@user.login, flash[:password], @site_url)
```

Comment:
I was going for a more functional approach. Maybe it doesn't work well with Ruby OO? A benefit is easier testing and easier reasoning about code because its more of a pure function. But consistency is probably more important, so I can change it to use ivars.